src/apps/material/material.tsx

```diff
@@ -194,25 +193,27 @@ const Material: React.FC<MaterialProps> = ({ wid }) => {
         )}
       </MaterialHeader>
       <MaterialDescription pid={pid} work={work} />
-      <MaterialDisclosure
-        title={`${t("editionsText")} (${manifestations.length})`}
-        icon={VariousIcon}
-        dataCy="material-editions-disclosure"
-      >
-        <>
-          {getManifestationsOrderByTypeAndYear(manifestations).map(
-            (manifestation: Manifestation) => {
-              return (
-                <MaterialMainfestationItem
-                  key={manifestation.pid}
-                  manifestation={manifestation}
-                  workId={wid}
-                />
-              );
-            }
-          )}
-        </>
-      </MaterialDisclosure>
+      {workType === "local" && (
```

Comment:
So we don't show editions for global materials? Maybe it would be nice with a comment explaining the logic behind this.